### ISSUE TITLE
[Desktop] Implement report an issue screen on desktop (already done on mobile)

### ISSUE THREAD

**Issue by ghost** [getlantern/lantern#4869]
Currently, the send log button is in the account management page, which is not accessible in Free. On mobile, it's a separate item in the drop down menu. We should do the same on desktop. 

@Derekf5 
The menu should say Report Issue. When a user click on the menu, Lantern will open a new windows. In this window, there will be a text box asking user to describe the issue/steps to reproduce the issue. Logs will also be automatically attached. System OS, Browser, will also be prefilled. Also ask for user's contact email.  And there will be a Send Report button.

**Comment by atavism** [comment]
Hey @Derekf5, could you design the UI for this? This is pretty high priority because we have lots of users submitting logs but no way to filter by issue or for the user to provide an e-mail for us to follow-up (for those users unable to sign-up for Pro).

**Comment by Derekf5** [comment]
@p3rcya if OS and browser is prefilled is there any reason to include it on the screen? cant we just automatically send it to ourselves with the logs?

**Comment by p3rcya** [comment]
@Derekf5 For desktop, it's fine if we just include it silently.
@atavism But for android, can those Chinese phone type be correctly automatically recognized? If so, then we should just include it silently. Otherwise, it's better to prefill and users can correct them

**Comment by carsonefv** [comment]
@percy let me know what you think of this, @chipstehr how does the copy on this report an issue page look to you?

Here are the changes needed for Desktop - 

1. Add "Report an Issue" to Menu for Free and Pro users:
![screenshot 2016-07-16 15 56 49](https://cloud.githubusercontent.com/assets/4400763/16896852/ed874eaa-4b6d-11e6-8613-9b18da3d1db4.png)

2. Pop-up Screen:
 Users should have to include an email to report an issue, so if it is left blank we should serve a "must include email" error. Users should also include a description, there should be an error in the text box saying "must provide description of issue" if users attempts to submit without filling in that field

Once the issue is succesfully submitted a snackbar should appear saying "Success, Thank you for reporting your issue."
![screenshot 2016-07-16 15 44 03](https://cloud.githubusercontent.com/assets/4400763/16896860/2755b5a4-4b6e-11e6-8932-74aaa3085803.png)
![screenshot 2016-07-16 16 12 15](https://cloud.githubusercontent.com/assets/4400763/16896916/2da3388a-4b70-11e6-909c-1744fb128753.png)

**Comment by p3rcya** [comment]
That looks good. But I don't know how important it is for users to attach screenshots for debugging @myleshorton @atavism If so, we can allow users to optionally attach images as well.

**Comment by carsonefv** [comment]
@p3rcya  Here's an update with drop-down menu of issues added
do we want to add the option to attach images/screenshots here too @myleshorton @fffw ?

the dropdown menu should work like this: https://material.angularjs.org/latest/demo/select

![screenshot 2016-07-19 12 42 26](https://cloud.githubusercontent.com/assets/4400763/16958610/30b969d6-4daf-11e6-9a0a-f86beb389879.png)
![screenshot 2016-07-19 12 42 39](https://cloud.githubusercontent.com/assets/4400763/16958615/36117f04-4daf-11e6-93a8-e909255605bc.png)

**Comment by p3rcya** [comment]
FYI, the type of issues we used on mobile are 
<item>Cannot access blocked sites</item>⏎
<item>Cannot complete purchase</item>⏎
<item>Cannot sign in</item>⏎
<item>App crashes</item>⏎
<item>Spinner loads endlessly</item>⏎
<item>Slow</item>⏎
<item>Other</item>⏎

**Comment by p3rcya** [comment]
It's applies to the desktop as well, with one minor change from App Crashes to Lantern Crashes.

**Comment by Derekf5** [comment]
can we get this into the next sprint @myleshorton?  it is already implemented on mobile

**Comment by p3rcya** [comment]
@Derekf5 @myleshorton This really really needs to be done before auto-update

**Comment by myleshorton** [comment]
@joesis is this actually in progress? is there a branch?

**Comment by fffw** [comment]
@myleshorton the changes are at the issue-4869 branch of both flashlight
and lantern-desktop-ui. What left is to wire UI with the code in flashlight to
collect logs.

**Comment by joesis** [comment]
The Lantern backend and desktop UI PR opened at https://github.com/getlantern/flashlight/pull/75 and https://github.com/getlantern/lantern-desktop-ui/pull/190 respectively
